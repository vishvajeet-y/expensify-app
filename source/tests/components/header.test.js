
import React from 'react'
import {shallow} from 'enzyme'
//import ReactShallowRendrer from 'react-test-renderer/shallow'
import Header from '../../components/Header'

test('should render Header correctly',()=>{

    const wrapper=shallow(<Header/>)
   // expect(wrapper.find('h1').text()).toBe('Expensify')
       expect(wrapper).toMatchSnapshot()
    // const renderer =new ReactShallowRendrer()
    // renderer.render(<Header/>)
    // expect(renderer.getRenderOutput()).toMatchSnapshot()
    //console.log(renderer.getRenderOutput())
})
