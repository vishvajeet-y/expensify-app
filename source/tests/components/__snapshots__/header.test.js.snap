// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/create"
  >
    Create Expense
  </NavLink>
</div>
`;
